### PR TITLE
remove create vaults and deposit buttons

### PR DESCRIPTION
**Pull request by sshmm**
## What
remove create vaults
remove deposit button
add hint banner

## Screenshots (if appropriate):

image
image

### CODE REVIEW

**Comment by crabsinger** [comment]
@sshmm jest is failing

File: cypress/e2e/createVault.cy.ts [hunk 1]
```diff
@@ -1,102 +0,0 @@
-import { deriveAccount, gqlQuery, injectWeb3 } from '../util';
```
Comment (crabsinger) [inline cypress/e2e/createVault.cy.ts:1]: cypress is currently disabled - do you wan tto try to reenable it?

Comment (sshmm) [reply cypress/e2e/createVault.cy.ts:1]: @crabsinger  ok I will try it

**Comment by wingfeatherwave** [comment]
minor copy update:
Screenshot 2024-06-04 at 2 34 23 PM

**Review by wingfeatherwave — APPROVED** [review]

**Pull request merged**